jmclient/jmclient/wallet.py

```diff
@@ -92,6 +93,17 @@ def estimate_tx_fee(ins, outs, txtype='p2pkh'):
         raise NotImplementedError("Txtype: " + txtype + " not implemented.")
 
 
+def get_default_locktime():
```

Comment:
Why is the function called `get_default_locktime()` ? What's default about it. The name doesn't make much sense IMO. Maybe `get_anti_fee_snipe_locktime()` or `compute_tx_locktime()` or `get_tx_locktime()`.